ros_ws/dev.launch

```diff
@@ -1,4 +1,40 @@
 <launch>
-    <node name="listener" pkg="auto_race_pg" type="listener" output="screen"/>
-    <node name="talker" pkg="auto_race_pg" type="talker" output="screen"/>
-</launch>
\ No newline at end of file
+    <!-- car control node -->
+    <node
+      respawn="true"
+      pkg="auto_race_pg"
+      type="car_control"
+      name="car_control"
+      output="screen" >
+    </node>
+
+    <!-- remote joy -->
+    <node
+      respawn="true"
+      pkg="auto_race_pg"
+      type="remote_joy"
+      name="remote_joy"
+      output="log" >
+    </node>
+
+    <!-- joy node -->
+    <node
+      respawn="true"
+      pkg="joy"
+      type="joy_node"
+      name="joy_node"
+      output="log" >
+      <param name="dev" type="string" value="/dev/input/js2" />
```

Comment:
Why js2? The default should probably be js0, maybe it can even be an argument in the roslaunch command.
```suggestion
      <param name="dev" type="string" value="/dev/input/js0" />
```